src/DGtal/helpers/Shortcuts.h

```diff
@@ -1651,6 +1653,67 @@ namespace DGtal
         else return surface->allVertices();
         return result;
       }
+      /// Outputs a digital surface as an OFF file (with its
+      /// topology).  Optionnaly you can specify the face colors (see
+      /// saveOBJ for a more advanced export).
+      ///
+      /// @tparam TDigitalSurfaceContainer any model of concepts::CDigitalSurfaceContainer
+      /// @tparam TCellEmbedder any type for maping Cell -> RealPoint.
+      ///
+      /// @param[in] digsurf the digital surface to output as an OBJ file
+      /// @param[in] embedder any map Cell->RealPoint
+      /// @param[in] face_colors either empty or a vector of size `trisurf.nbFaces` specifying the diffuse color for each face.
+      /// @param[in] offfile the output filename.
+      /// @return 'true' if the output stream is good.
+      template <typename TDigitalSurfaceContainer,
+        typename TCellEmbedder>
+        static bool
+        saveOFF
+        ( CountedPtr< ::DGtal::DigitalSurface<TDigitalSurfaceContainer> > digsurf,
+          const TCellEmbedder&           embedder,
+          std::string                    off_file,
+          const Color&                   face_color=DGtal::Color::None)
+
+        {
+          BOOST_STATIC_ASSERT (( KSpace::dimension == 3 ));
+
+          std::ofstream output_off( off_file.c_str() );
+          output_off << "OFF" << std::endl;
+          output_off << "# Generated from  DGtal::Shortcuts from the DGTal library" << std::endl;
+          Cell2Index      c2i;
+          auto       pointels = getPointelRange( c2i, digsurf );
+          output_off <<  pointels.size()  << " " << digsurf->size() << " " << 0 << " " << std::endl;
+          
+          
+          // Number and output vertices.
+          const KSpace&     K = refKSpace( digsurf );
+          for ( auto&& pointel : pointels )
+            {
+              RealPoint p = embedder( pointel );
+              output_off << p[ 0 ] << " " << p[ 1 ] << " " << p[ 2 ] << std::endl;
+            }	
+          
+          // Taking care of faces
+          for ( auto&& surfel : *digsurf )
+            {              
+              auto primal_vtcs = getPointelRange( K, surfel );
+              output_off << primal_vtcs.size();
+                {
+                  for ( auto&& primal_vtx : primal_vtcs )
+                    output_off << " " << (c2i[ primal_vtx ]);
+                }
+                if(face_color != DGtal::Color::None)
+                {
+                  output_off << " ";
+                  output_off  << ((double) face_color.red())/255.0 << " "
```

Comment:
We should add methods r() g() b() a() in Color to return a double in [0,1]

Reply:
yes done